.ado/variables/vars.yml

```diff
@@ -0,0 +1,6 @@
+variables:
+  VmImageApple: macOS-11
+  VmImageWindows: windows-2019
+  VmImageUbuntu: ubuntu-20.04
+  slice_name: 'Xcode_13_1'
+  xcode_version: '/Applications/Xcode_13.1.app'
```

Comment:
@HeyImChris was previously working on getting this repo on Xcode 13, and I think ran into issues? He'll probably want to comment. We care a lot about keeping Xcode version used in react-native-macos in sync with downstream, so this would be great!

Reply:
I've used Xcode 13 without issues against this repo for the RNtester, which is why I did this bump - what were the problems hit?

Reply:
I think it was mostly with downstream integration into SDX-Platform and the like. I'm not too sure.

Reply:
I think we're good to go forward with Xcode 13 and Big Sur support now! I created a PR today doing just this but then Saad reminded me of your review. Let's just get this in soon so our CI is up to date - @kelset let me know if you need help with the CI or anything else. It can be a bit flaky at times